### PR TITLE
Added recipe for nutch

### PR DESCRIPTION
**Pull request by pmlandwehr**
This is a recipe for the python bindings for the [Nutch web crawler](https://nutch.apache.org/)

### CODE REVIEW

File: recipes/nutch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  imports:
+    - nutch
+
+  commands:
+    - nutch-python --help
+
+about:
+  home: https://github.com/chrismattmann/nutch-python
+  license_file: LICENSE.txt
+  license: Apache
```
Comment (synapticarbors) [inline recipes/nutch/meta.yaml]: Change to `Apache-2.0`

File: recipes/nutch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
```
Comment (synapticarbors) [inline recipes/nutch/meta.yaml]: Is this required? The docs on this are not very insightful:

https://conda.io/docs/building/meta-yaml.html#preserve-python-egg-directory

File: recipes/nutch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+
+  run:
+    - python
+    - setuptools
+    - requests
```
Comment (synapticarbors) [inline recipes/nutch/meta.yaml]: Are we just assuming that the user has their own version of nutch? The conda recipe in the upstream repo makes it a requirement:

https://github.com/chrismattmann/nutch-python/blob/master/conda.recipe/meta.yaml#L45

File: recipes/nutch/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{% set name = "nutch" %}
+{% set version = "1.10.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "17cf195db88b11c5d283448fe362d483e78db58dc564339583de629790103782" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
```
Comment (jakirkham) [inline recipes/nutch/meta.yaml]: Might want to do a `python-` prefix here as this is a Python client to Apache Nutch. Guessing we want our own copy of Apache Nutch down the road.

**Comment by scopatz** [comment]
pinging this!

**Pull request closed without merging**